docker-compose.yml

```diff
@@ -15,18 +15,18 @@ services:
       - "7777:7777"
     environment:
       - RUN_DEV_SERVER=True
-      - PORT=7777
-      - HOST=0.0.0.0
       - DEBUG=True
+      - PROTOCOL=https
```

Comment:
SSO Dashboard does this:

https://github.com/mozilla-iam/sso-dashboard/blob/master/dashboard/app.py#L155

Thoughts?